README.md

```diff
@@ -200,7 +202,7 @@ field is a free-form string that identifies the build and test runner.
 ### Robot Comments
 
 Robot comments are comments generated by static analysis tools. These are
-stored in the "refs/notes/devtools/analyses" ref, and annotate the revision.
+stored in the "refs/notes/git-review/robot" ref, and annotate the revision.
```

Comment:
Yeah, you're right about that; referring to them as "Analyses Results" in the docs would be better than "Robot Comments".

I should create a bug to update the docs accordingly.